src/building/how-to-build-and-run.md

```diff
@@ -40,10 +40,15 @@ of a checkout. It also looks up the appropriate version of `python` to use.
 
 You can install it with `cargo install --path src/tools/x`.
 
-## Create a `config.toml`
+## Configuring rustbuild
 
-To start, run `./x.py setup`. This will do some initialization and create a
-`config.toml` for you with reasonable defaults.
+rustbuild offers a TOML-based configuration system with a `config.toml`
+file. An example of this configuration can be found at `config.toml.example`,
+and the configuration file can also be passed as `--config path/to/config.toml`
+if the build system is being invoked manually (via the python script).
+
+You can generate a config.toml using `./configure` or `./x.py setup`
+options if you want to automate creating the file without having to edit it.
```

Comment:
Please revert this, it's too much detail for the getting started section. I intentionally had a single line people could copy paste as the begging of the section.

Reply:
Reverted